src/core/mount.c

```diff
@@ -1883,8 +1952,6 @@ static int mount_load_proc_self_mountinfo(Manager *m, bool set_flags) {
                 if (!device || !path)
                         continue;
 
-                device_found_node(m, device, DEVICE_FOUND_MOUNT, DEVICE_FOUND_MOUNT);
```

Comment:
if all you do is prep a set of devices to later operate on, why not just do so here, on the fly?